### PR TITLE
deepCopy Function

### PR DESCRIPTION
**Pull request by ThinkTankShark**
- Adds deepCopy function to utility library
- Unit tests for deepCopy
- Updates readme file and package.json

### CODE REVIEW

File: src/__tests__/utils-spec.js [hunk 1]
```diff
@@ -178,6 +178,110 @@ describe('Utils Library', () => {
     });
   });
 
+  describe('deepCopy', function(){
```
Comment (isuttell) [inline src/__tests__/utils-spec.js]: We should test if it's actually a copy or a reference as well.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ship-components-utility",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (isuttell) [inline package.json]: This should probably be 1.1.0 according to the [semver](http://semver.org/) spec. So, let's switch that.

> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

File: src/__tests__/utils-spec.js (hunk 1 above)
Comment (ThinkTankShark) [inline src/__tests__/utils-spec.js]: Added.

File: package.json (hunk 2 above)
Comment (ThinkTankShark) [inline package.json]: updated.

File: src/utils.js [hunk 3]
```diff
@@ -127,18 +127,61 @@ export function mergeDeep(target, source) {
...
+  var result;
+
+  // Handle Date
+  if (obj instanceof Date) {
+    result = new Date();
+    result.setTime(obj.getTime());
+    return result;
+  }
+
+  var gdcc = '__getDeepCircularCopy__';
```
Comment (isuttell) [inline src/utils.js]: What is this?

File: src/utils.js [hunk 4]
```diff
@@ -127,18 +127,61 @@ export function mergeDeep(target, source) {
  * @param     {Array<Objects> || Objects}    obj
  * @return    {Array<Objects> || Objects}
  */
+/* eslint-disable */
```
Comment (isuttell) [inline src/utils.js]: We shouldn't be disabling linting. Add it back in please.

**Pull request merged**